src/pages/iou/IOUCurrencySelection.js

```diff
@@ -43,10 +45,19 @@ const propTypes = {
         currency: PropTypes.string,
     }),
 
+    /** The report on which the request is initiated on */
+    report: reportPropTypes,
+
+    /** Any errors associated with an attempt to create a chat */
+    // eslint-disable-next-line react/forbid-prop-types
+    errors: PropTypes.object,
+
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
+    report: {},
+    errors: {},
```

Comment:
Coming from https://github.com/Expensify/App/issues/23373

The `errors` props are not passed from anywhere in this component. Also, note for me to make sure the props define are necessary for the component and used by their parent.